Branch-SDK/BNCPreferenceHelper.h

```diff
@@ -29,9 +29,10 @@ NSURL* /* _Nonnull */ BNCURLForBranchDirectory(void);
 @property (copy, nonatomic) NSString *sessionID;
 @property (copy, nonatomic) NSString *linkClickIdentifier;
 @property (copy, nonatomic) NSString *spotlightIdentifier;
-@property (copy, nonatomic)    NSString *universalLinkUrl;
-@property (copy, nonatomic)    NSString *initialReferrer;
+@property (copy, nonatomic) NSString *universalLinkUrl;
+@property (copy, nonatomic) NSString *initialReferrer;
 @property (copy, nonatomic) NSString *userUrl;
+@property (copy, nonatomic) NSString *localUrl;
```

Comment:
I think we need to track whether we've sent this value up. BNCPreferenceHelper is persistent, so setting it here without later clearing it means we send it up with subsequent opens.

Reply:
We could also just keep it in memory instead of saving it to preference helper, but that could have other side effects.

Reply:
@echo-branch OK. Will clear it after first successful open. I thought it will be useful to send it with every open.

Reply:
@echo-branch I have added check for, sending local_url only on first open after data is copied, here in this commit https://github.com/BranchMetrics/ios-branch-deep-linking-attribution/pull/1199/commits/3beb379491320c691ed0749afd6a48042c672eeb